README.md

```diff
@@ -85,6 +98,8 @@ sudo sed -i -e 's/SELINUX=enforcing/SELINUX=permissive/' /etc/selinux/config
 sudo dnf install -y --enablerepo=updates-testing osbuild osbuild-tools osbuild-ostree podman jq xfsprogs e2fsprogs
 ```
 
+Remember to reboot the system after setting SELinux to permissive mode.
+
```

Comment:
not necessary. `sudo setenforce 0` is applying it to the live system and `sudo sed -i -e 's/SELINUX=enforcing/SELINUX=permissive/' /etc/selinux/config` is making it apply for subsequent boots.

Reply:
Understood, thank you.